client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.html

```diff
@@ -3,124 +3,126 @@
-                        data-ng-if="!$row.$last"
-                        data-ng-click="$ctrl.moveLineOrder($row, 'down')">
-                    <span class="oui-icon oui-icon-chevron-down"
-                          aria-hidden="true"></span>
-                </button>
-            </oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_type' | translate"
-                        data-property="type">
-                <span data-translate="{{:: 'telephony_alias_config_contactCenterSolution_lines_type_' + $value }}"></span>
-            </oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_number' | translate"
-                        data-property="number">
-                <span data-ng-bind="::$value | tucPhoneNumber"></span>
-            </oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_name' | translate"
-                        data-property="description"></oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_timeout' | translate"
-                        data-property="timeout">
-                <span data-translate="telephony_alias_config_contactCenterSolution_lines_seconds"
-                      data-translate-values="{ 'amount': $value }"></span>
-            </oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_wrapUpTime' | translate"
-                        data-property="wrapUpTime">
-                <span data-translate="telephony_alias_config_contactCenterSolution_lines_seconds"
-                      data-translate-values="{ 'amount': $value }"></span>
-            </oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_simultaneous_lines' | translate"
-                        data-property="simultaneousLines"></oui-column>
-            <oui-column data-title="'telephony_alias_config_contactCenterSolution_lines_status' | translate"
-                        data-property="status">
-                <span data-translate="{{:: 'telephony_alias_config_contactCenterSolution_lines_status_' + $value }}"></span>
-            </oui-column>
-            <oui-action-menu data-align="end"
-                             data-compact>
-                <oui-action-menu-item data-text="{{:: 'modify' | translate }}"
-                                      data-on-click="$ctrl.updateLine($row)"></oui-action-menu-item>
-                <oui-action-menu-item data-text="{{:: 'delete' | translate }}"
-                                      data-on-click="$ctrl.deleteLineOpenModal($row)"></oui-action-menu-item>
-            </oui-action-menu>
-        </oui-datagrid>
+        <div class="mt-5"
+             data-ng-if="!$ctrl.updatingAgents && $ctrl.agents.length > 0">
+            <oui-datagrid class="d-md-flex row-height-limited"
```

Comment:
Shouldn't be handled by the `<oui-datagrid>` itself? I mean (.d-md-flex.row-height-limited)

Reply:
`d-md-flex` is not necessary
`row-height-limited` is for the moment scoped into telecom only, because the oui-datagrid component is not intended for handle line order (purpose of these two buttons linked to `row-height-limited` class)